stable/mariadb/values.schema.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "rootUser": {
+      "type": "object",
+      "properties": {
+        "password": {
+          "type": "string",
+          "title": "MariaDB admin password",
+          "form": true,
+          "description": "Defaults to a random 10-character alphanumeric string if not set",
+          "hidden": {
+            "condition": false,
+            "value": "usePassword"
+          }
+        }
+      }
+    },
+    "db": {
+      "type": "object",
+      "properties": {
+        "user": {
+          "type": "string",
+          "title": "MariaDB custom user",
+          "form": true
+        },
+        "name": {
+          "type": "string",
+          "title": "MariaDB custom database",
+          "form": true
+        },
```

Comment:
can you add a small description for these two parameters? (so it's clear what it does)